guides/micronaut-custom-validation-annotation/java/src/test/java/example/micronaut/ContactTest.java

```diff
@@ -0,0 +1,44 @@
+package example.micronaut;
+
+import io.micronaut.context.MessageSource;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validator;
+
+import java.util.Locale;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@MicronautTest(startApplication = false) // <3>
```

Comment:
Should be 1?

Reply:
https://github.com/micronaut-projects/micronaut-guides/pull/1244/commits/dba086824f1276407f6da8b0e04f8c028d673f85